<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BootOptionsEditPage</class>
 <widget class="QWidget" name="BootOptionsEditPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBoxAutoBoot">
     <property name="text">
      <string>Start VM automatically on server boot</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxBootOrder">
     <property name="title">
      <string>Boot Order (HVM VMs only)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="labelBootOrderInfo">
        <property name="text">
         <string>Specify the order in which the VM attempts to boot from devices.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QListWidget" name="listWidgetBootOrder">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="buttonUp">
            <property name="text">
             <string>Move Up</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonDown">
            <property name="text">
             <string>Move Down</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="buttonConvertToPV">
        <property name="text">
         <string>Convert to PV</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxPVParams">
     <property name="title">
      <string>PV Boot Parameters</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelPVBootDevice">
        <property name="text">
         <string>Boot device:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxBootDevice"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelOsParams">
        <property name="text">
         <string>OS Boot Parameters:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditOsParams">
        <property name="placeholderText">
         <string>e.g., console=hvc0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="buttonConvertToHVM">
        <property name="text">
         <string>Convert to HVM</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
